Pear/Profile/EditProfileSectionsViewController.swift

```diff
@@ -145,3 +230,66 @@ extension EditProfileSectionsViewController: UITableViewDataSource {
 extension EditProfileSectionsViewController: UITableViewDelegate {
     
 }
+
+extension EditProfileSectionsViewController: didUpdateInterestsDelegate, didUpdateGroupsDelegate, didUpdatePromptsDelegate {
```

Comment:
So yeah I think here instead of conforming to 3 or 4 protocols, just create 1 protocol with delegate methods for updating groups, interests, etc.